libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_WIN.c

```diff
@@ -1946,7 +1998,7 @@ const int32_t l500ms = 500;
 				}
 
 				/* Nothing to send right now. */
-				ulDataLength = 0ul;
+				ulDataLength = 0UL;
```

Comment:
Again, sorry for adding this comment here. I cannot comment on line 2026. The function there should be `BaseType_t xTCPWindowRxEmpty( TCPWindow_t *pxWindow )` in accordance with the new declaration in the FreeRTOS_TCP_WIN.h

Reply:
Yes, same thing as with `xTCPWindowTxDone()`, a const must be added. 
I should have compiled with and without the macro `ipconfigUSE_TCP_WIN`.